src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -56,6 +64,13 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
 
     private activeFoci: Focus[] | undefined;
 
+    private encryptMedia = false;
+    private _encryptionKeys = new Map<{ userId: string; deviceId: string }, string>();
+
+    public get encryptionKeys(): Map<{ userId: string; deviceId: string }, string> {
```

Comment:
I don't really see the advantage of making the variable private but then just exposing it with a getter. I think my 1st choice would be to hide the map entirely with `getEncryptionKey()` & `setEncryptionKey()` functions. If that's too much boilerplate, `public readonly encryptionKeys  = ...` would be the same as this and shorter I think.

Reply:
So the `get encryptionKeys()` method actually returns a copy of the map which the `readonly` approach doesn't, afaik. I also want EC to be able to access all keys we have using one method/getter

Reply:
This still seems like a really roundabout way of achieving this to me, and clones map unnecessarily. What do you mean by, "access all keys we have", as in encryptionsKeys.values()? What's the value in it all being one getter? `getKey()` / `getAllKeys()` seems simpler & more explicit to me.